### PR TITLE
choice, net, sc, w32tm: remove locale from link; sc: update page, add alias pages

### LINKED ISSUE #11229: page modification request: remove locale from link
### Command description

In pages/windows there are four pages that include `en-us` in the url. We don’t include a locale in the url as described [here](https://github.com/tldr-pages/tldr/pull/10963#pullrequestreview-1677793645). The four pages are:
- [ ] [sc](https://github.com/tldr-pages/tldr/blob/015e62b12817ab9d712a266a47b116ba9d3018c0/pages/windows/sc.md?plain=1#L4)
- [ ] [net](https://github.com/tldr-pages/tldr/blob/015e62b12817ab9d712a266a47b116ba9d3018c0/pages/windows/net.md?plain=1#L4)
- [ ] [w32tm](https://github.com/tldr-pages/tldr/blob/015e62b12817ab9d712a266a47b116ba9d3018c0/pages/windows/w32tm.md?plain=1#L4)
- [ ] [choice](https://github.com/tldr-pages/tldr/blob/015e62b12817ab9d712a266a47b116ba9d3018c0/pages/windows/choice.md?plain=1#L4)

Also please check if the pages are still up-to-date.

### OS

- Windows

### PR DESCRIPTION
**Pull request by deadpyxel**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

This should fix #11229

### CODE REVIEW

File: pages/windows/sc.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # sc
 
 > Communicate with the Service Control Manager and services.
-> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/sc-query>.
+> More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/sc-query>.
```
[the remark this answers is not in the document]
Comment (deadpyxel) [reply pages/windows/sc.md:4]: Indeed, I am updating it right now.

Also, currently, the description says:

> Communicate with the Service Control Manager and services.

But their docs are saying:

> Obtains and displays information about the specified service, driver, type of service, or type of driver.

Also, the link points the the `sc-query` command, and as such maybe this should be renamed? I don't have a windows VM handy right now to test the commands being mentioned on the docs (there's no `start` for example). Should this be renamed?

**Comment by deadpyxel** [comment]
I think I got what is happening now. The command being listed by us on tldr has multiple entries on their docs, but we are linking only one. As we only link one documentation page, should we keep it as is or change it to point to `sc-config`?

File: pages/windows/sc.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (deadpyxel) [reply pages/windows/sc.md:4]: Applied updates on cb613f5

File: pages/windows/net.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # net
 
 > System utility to view and modify network-related settings.
-> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/gg651155(v=ws.11)>.
+> More information: <https://learn.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/gg651155(v=ws.11)>.
```
[the remark this answers is not in the document]
Comment (deadpyxel) [reply pages/windows/net.md:4]: Interestingly, this link is working for me, I am being redirected to the en-gb locale.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
> I don't know what other contributors think, but in my opinion we should keep the file as is for now and only update the commands to reflect the 4 pages in one tldr page.

Updating the `sc` page has been on my to-do list for a while (since I merged the previous update to the page [Made a comment about it there too]). I think we can create alias pages for the subcommands to reference to the main page [I will do it now].

**Review by kbdharun — APPROVED** [review]

**Pull request merged**